LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -245,8 +275,12 @@ public void AssignInstructorsToClassrooms()
                     {
                         ModuleInstructorListDictionary.TryGetValue(moduleId, out instructorList);
                         classroom.Head = instructorList[AttributeGenerator.random.Next(0, instructorList.Count)];
+                        ReportItem(itemName: classroom.ToString(), actionName: "Instructor assignment", ++count);
                     }
```

Comment:
Add an **else** block to report the error: no instructors for the classroom

Reply:
Done